changelog.md

```diff
@@ -30,7 +30,7 @@ This changelog is based on [Keep A Changelog](https://keepachangelog.com/en/1.1.
 * polkadot-sdk dependency updated to partnerchains-stable2407 (stable2407 is v1.15.0 in the previous scheme)
 * changed the inner type of `McBlockHash` from Vec to an array
 * ETCM-7985 - bumped `partner-chains-smart-contracts` version and updated the `partner-chains-cli`
-to match. Now `partner-chains-cli` passes a network parameter to `sidechain-main-cli` where necessary.
+to match. Now `partner-chains-cli` passes a network parameter to `pc-contracts-cli` where necessary.
```

Comment:
I disagree with modification of past changes. I think, that instead this current change should be added on top of it.

Reply:
thanks, it was an overlook